cartridges/int_bolt_embedded_sfra/cartridge/controllers/CheckoutServices.js

```diff
@@ -3,11 +3,6 @@
 /* API Includes */
 var server = require('server');
 
-// Script includes
-var boltAccountUtils = require('~/cartridge/scripts/util/boltAccountUtils');
-var logUtils = require('~/cartridge/scripts/util/boltLogUtils');
-var log = logUtils.getLogger('CheckoutService');
-
 server.extend(module.superModule);
 
 server.append('PlaceOrder', function (req, res, next) {
```

Comment:
should we log error if res.viewData.error is true or res.viewData is null?

Reply:
I think that's covered in base cartridge and it wouldn't get here if there is an error